e2e-tests/tests/plan-simulation-templates.test.ts

```diff
@@ -75,19 +75,19 @@ test.describe.serial('Plan Simulation Templates', async () => {
   test(`Setting a simulation template to a simulation should update the parameter values`, async () => {
     await plan.selectSimulationTemplateByName('Template 1');
 
-    expect(plan.panelSimulation.getByRole('textbox', { name: 'Template 1' })).toBeVisible();
+    expect(plan.panelSimulation.getByRole('combobox', { name: 'Template 1' })).toBeVisible();
   });
 
   test(`Removing an simulation template from a simulation should reflect that it is no longer present`, async () => {
     await plan.selectSimulationTemplateByName('None');
 
-    expect(page.getByRole('textbox', { name: 'None' })).toBeVisible();
+    expect(page.getByRole('combobox', { name: 'None' })).toBeVisible();
   });
 
   test('Deleting an simulation template should remove it from the list of templates', async () => {
     await plan.selectSimulationTemplateByName('Template 1');
 
-    await page.getByRole('button', { name: 'Set Template' }).click();
+    await plan.panelSimulation.locator('div[name="Set Template"]').click();
```

Comment:
It's no longer possible to get this by role?

Reply:
Reverted and updated to the correct role